docs/modules/events/pages/object-events.adoc

```diff
@@ -208,6 +208,8 @@ Its default value is `false`.
 You can listen to entry-based events in the MultiMap using `EntryListener`.
 The following is an example entry listener implementation for MultiMap.
 
+NOTE:  MultiMap Events support only `entryAdded`, `entryUpdated` and `mapCleared` events.
```

Comment:
May. be rewriting like the following?

The entry listener for MultiMap supports only .......

Reply:
Sure.